packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    &__grid {
+      @include carbon--make-container;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__left,
+    &__children {
+      @include carbon--make-col-ready;
+    }
+    &__left {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 16);
+      }
+    }
+    &__heading {
+      margin-bottom: $carbon--layout-05;
+      @include type-style('expressive-heading-02');
+    }
+    &__children {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(12, 16);
+      }
+    }
+
+    padding-top: $carbon--layout-01;
+    padding-bottom: $carbon--layout-05;
+
+    @include carbon--breakpoint('lg') {
+      &__heading {
+        padding-top: $carbon--layout-03;
```

Comment:
I think you might be adding too much spacing here on `lg`. I don't think this `padding-top` is necessary.

Reply:
I also think it's getting overridden from the one below it.